### ISSUE TITLE
Action Popovers: Focus

### ISSUE THREAD

**Issue by bmrgould** [gctools-outilsgc/gccollab-mobile#303]
Home and Group pages have an action popover to post multiple types of content to the site/group. When opening this popover the focus should be set to a new popover label for Screen Reader/ Non Visual User context.
